### PR TITLE
risk exposure overview

### PR DESCRIPTION
**Pull request by yongqx**
Initialize risk exposure doc with GBD overview

### CODE REVIEW

File: docs/source/models/risk.rst [hunk 1]
```diff
@@ -4,17 +4,100 @@
...
+   - e.g. burden of lung cancer due to smoking
+- Comparative importance of the underlying risk factors which cause disease and
+  injuries
+   - e.g. burden of smoking compared with unsafe water
+
+How do we measure?
+++++++++++++++++++
+Input:
+ - Risk factor exposure (current distribution and counterfactual distribution)
+ - Risk factor-disease relationship (risk accumulation and risk reversal)
```
Comment (aflaxman) [inline docs/source/models/risk.rst]: Is this your wording or something from a GBD report?  The parenthetical terms are not familiar to me.

Comment (yongqx) [reply docs/source/models/risk.rst]: Yes, this paragraph came from old GBD risk factor training slides. Do you suggest we find update-to-date GBD doc and rephrase it to sim science language?

File: docs/source/models/risk.rst [hunk 2]
```diff
@@ -4,17 +4,100 @@
...
+GBD Overview
+------------
 
 What is a risk factor?
-----------------------
+++++++++++++++++++++++
+A risk factor is any attribute, characteristic or exposure of an individual 
+that increases the likelihood of developing a disease or injury. Some examples 
+of the more important risk factors are underweight, unsafe sex, high blood pressure, tobacco and alcohol consumption, sanitation and hygiene, and unsafe water. [WHO]_ In GBD, a risk factor refers to any exposure that leads to a 
+loss of health in the population; it can be anywhere on the causal chain as
```
Comment (alibow) [inline docs/source/models/risk.rst]: I would change "In GBD, a risk factor refers to **any** exposure that leads to a loss of health" to "In GBD, a risk factor refers to **a** exposure that leads to a loss of health" because the GBD list of risk factors is not an exhaustive list of risks that lead to health loss (this may be worth noting!)

File: docs/source/models/risk.rst [hunk 3]
```diff
@@ -4,17 +4,100 @@
...
+long as there is evidence to show that the risk factor is causally linked to an outcome disease or injury.
+
+What do we measure?
++++++++++++++++++++
+- Proportion of disease and injury burden attributable to specific risk factors
+   - e.g. burden of diarrheal due to unsafe water
+   - e.g. burden of lung cancer due to smoking
+- Comparative importance of the underlying risk factors which cause disease and
+  injuries
+   - e.g. burden of smoking compared with unsafe water
```
Comment (alibow) [inline docs/source/models/risk.rst]: I would specifically name 'population attributable fraction' and 'summary exposure value' as the terms for these definitions here.

File: docs/source/models/risk.rst [hunk 4]
```diff
@@ -4,17 +4,100 @@
...
+
+How do we measure?
+++++++++++++++++++
+Input:
+ - Risk factor exposure (current distribution and counterfactual distribution)
```
Comment (alibow) [inline docs/source/models/risk.rst]: It would be good to clarify what the counterfactual distribution means here!

File: docs/source/models/risk.rst (hunk 2 above)
Comment (yongqx) [reply docs/source/models/risk.rst]: Thanks for pointing it out! The GBD 2019 risk factor list lives at: https://hub.ihme.washington.edu/display/GBD2019/GBD+2019+Risk+factors?preview=/54736328/80467412/list_of_risks_gbd_2019.xlsx

File: docs/source/models/risk.rst (hunk 1 above)
Comment (aflaxman) [reply docs/source/models/risk.rst]: Let's change it to

  - Risk factor exposure (e.g. prevalence for dichotmous risks; mean, standard deviation, ensemble weights for continuous risks with an ensemble distribution)

  - Risk factor-disease relationship (relative risk of disease outcome at each level of risk factor exposure)

Comment (yongqx) [reply docs/source/models/risk.rst]: Will do!

File: docs/source/models/risk.rst [hunk 5]
```diff
@@ -4,17 +4,148 @@
...
+------------
 
 What is a risk factor?
-----------------------
+++++++++++++++++++++++
+A risk factor is any attribute, characteristic or exposure of an individual
+that increases the likelihood of developing a disease or injury. In GBD, a risk factor refers to a exposure that leads to a loss of health in the population;
+it can be anywhere on the causal chain as long as there is evidence to show that the risk factor is causally linked to an outcome disease or injury.
+
+What do we measure?
```
Comment (alibow) [inline docs/source/models/risk.rst]: I think we should also consider 1) exposure of a risk (absolute, not a SEV) and 2) relationship between risk and outcome pairs (risk ratios) as things that are measured in GBD

**Comment by beatrixkh** [comment]
From documentation session:
here is template for risk exposure modeling pages james organized for us:
	- Overview (what is risk exposure)
	- Exposure types (examples)
		○ Dichotomous
		○ Ordinal
		○ Unordered
		○ Bivariate
		○ Normal
		○ Lognormal
		○ Ensemble
	- Data sources
	- Vivarium models
	- TMRL
        - V&V

Note that this only maps out the 'exposure' part of risk factor modeling. We haven't yet planned out how to document RR and PAFs (which are typically the other 2 components of a risk factor model within GBD)

As this structure is quite different, we're going to close this PR and use it as a reference to flesh out the pages listed above.

Anyone who has time: feel free to start on any of the above pages, please do so and add it to jira. Thanks!
